src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy

```diff
@@ -190,15 +185,7 @@ class Liberty implements Plugin<Project> {
             description "Copy applications generated by the Gradle project to a Liberty server's dropins or apps directory."
             logging.level = LogLevel.INFO
             group 'Liberty'
-            dependsOn project.tasks.withType(War)
-
-            project.afterEvaluate {
-                if (server.features.name != null && !server.features.name.isEmpty()) {
-                    dependsOn 'installFeature'
-                } else {
-                    dependsOn 'libertyCreate'
-                }
-           }
+            dependsOn project.tasks.withType(War), 'libertyCreate'
```

Comment:
remove for uninstallFeature

Reply:
uninstallFeature is clean. leaving this line under installApps

Reply:
I think that the code must have been collapsed and I thought it was under uninstall.